beginner_source/introyt/introyt1_tutorial.py

```diff
@@ -304,7 +304,8 @@ def num_flat_features(self, x):
 #    standard deviations (second tuple) of the rgb values of the images in
 #    the dataset. You can calculate these values yourself by running these
 #    few lines of code:
-#          ```
+#          ::
```

Comment:
This looks good but I would remove the extra spaces that are causing it to indent the code block, it makes it harder to read when the block isn't as wide.

Reply:
totally agree -> we should indent to the left for proper rendering